migrations/20240304163225_users.sql

```diff
@@ -5,7 +5,8 @@ CREATE TABLE IF NOT EXISTS users (
     name VARCHAR(255) NOT NULL,
     age VARCHAR(255) NOT NULL,
     gender VARCHAR(255) NOT NULL,
-    phone VARCHAR(255) NOT NULL
+    phone VARCHAR(255) NOT NULL,
+    role VARCHAR(255) NOT NULL
```

Comment:
this change should be in a new migration, after a migration is deployed it should not be changed